examples/react-spa/src/Settings.tsx

```diff
@@ -77,7 +95,7 @@ export const Settings = () => {
       getFlow(flowId).catch(createFlow) // if for some reason the flow has expired, we need to get a new one
       return
     }
-    createFlow()
+    createFlow().catch((error) => console.error(error))
```

Comment:
not sure if we can catch any errors here since `createFlow` doesn't return the error in the first catch clause.